.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: node_js
 node_js:
 - '10'
-- '9'
 - '8'
-- '7'
 - '6'
-- '5'
-- '4'
-- '0.12'
-- '0.10'
+
+env:
+  matrix:
+    - PACKAGE = ci-info
+    - PACKAGE = is-ci
+    - PACKAGE = is-pr
+
+branches:
+  only:
+  - master
+
+install:
+  - yarn
```

Comment:
This is temporary. Will rollback to default before merge. like u had before `No lockfile`.

Reply:
> I also like it if the Travis CI runs with the latest dependencies and sub-dependencies all the time, as that mimics what users of the module will experience

Point.

> There's probably and easy way to get Travis to ignore them though, I just haven't spend any time researching this .

I haven't done it either but Just an thought. If we specify the package-manager other than lock file explicitly for travis. It will regenerate lock files with whatever latest dependencies.

Simply, If we use npm, use yarn for travis and vice versa.

Using npm or yarn mostly doesn't matter in travis, for now.

Reply:
I'm fine with leaving the yarn lock file in the repo then and just using npm on travis then. Is that ok?

Reply:
its fine too.